src/component/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
+  <section className="section">
+    <p className="box is-size-3">
+      Please get your
+      {' '}
```

Comment:
don't create spaces this way, use the margins and padding instead 
check other places